extension/chrome/elements/pgp_block_modules/pgp-block-attachmens-module.ts

```diff
@@ -100,7 +100,7 @@ export class PgpBlockViewAttachmentsModule {
   };
 
   private isAttachmentURLInvalid = (attachment: Attachment): boolean => {
-    return new URL(String(attachment.url)).host !== 'flowcrypt.s3.amazonaws.com';
+    return attachment.url !== undefined && new URL(attachment.url).host !== 'flowcrypt.s3.amazonaws.com';
```

Comment:
`attachment.url !== undefined` can be shortened to `!attachment.url` (which also checks empty string)

Reply:
Thank you - I learned something new here.